src/main/java/core/basesyntax/service/operation/impl/OperationBalanceHandler.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.service.operation.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.operation.OperationHandler;
+
+public class OperationBalanceHandler implements OperationHandler {
+
+    @Override
+    public void operationHandler(FruitTransaction fruitTransaction) {
+        operationDao.put(fruitTransaction.getFruit(),fruitTransaction.getQuantity());
```

Comment:
Missing space after the comma in the method call, it should be formatted as `operationDao.put(fruitTransaction.getFruit(), fruitTransaction.getQuantity());`